integration/spark2/src/main/scala/org/apache/carbondata/spark/rdd/CarbonDataRDDFactory.scala

```diff
@@ -205,7 +205,8 @@ object CarbonDataRDDFactory {
 
               val newCarbonLoadModel = prepareCarbonLoadModel(table)
 
-              val compactionSize = CarbonDataMergerUtil.getCompactionSize(CompactionType.MAJOR)
+              val compactionSize = CarbonDataMergerUtil
+                .getCompactionSize(CompactionType.MAJOR, carbonLoadModel)
```

Comment:
why need add one more input parameter(carbonLoadModel) for getCompactionSize()?

Reply:
carbonLoadModel may contain table-level major compaction size if it is specified in create table SQL, so the purpose for adding parameter 'carbonLoadModel' is to get the table-level major compaction size.

Reply:
ok